api/desecapi/templates/emails/captcha/subject.txt

```diff
@@ -0,0 +1 @@
+deSEC dedyn.io Account Suspended
```

Comment:
will be using the 'first' domain name of the user at hand, and 'deSEC dedyn.io' if no domain name given.